github_plugin/mapping.py

```diff
@@ -9,14 +9,17 @@
     FieldDisplayIcon
 )
 
-from external_plugins.zendesk_plugin import transformer_functions
-import external_plugins.zendesk_plugin.default as DEFAULT
+from external_plugins.github_integrate import transformer_functions
```

Comment:
same here change the github_integrate  to github_plugin

Reply:
done